news/_posts/core-weekly/2019-05-31-coreweekly-week-21-2019.md

```diff
@@ -0,0 +1,147 @@
+- [77 issues have been closed](https://github.com/search?q=org%3APrestaShop+is%3Apublic++-repo%3Aprestashop%2Fprestashop.github.io++is%3Aissue+closed%3A2019-05-20..2019-05-26), including [18 fixed issues](https://github.com/search?q=org%3APrestaShop+is%3Apublic++-repo%3Aprestashop%2Fprestashop.github.io++is%3Aissue+label%3Afixed+closed%3A2019-05-20..2019-05-26) on the core;
+- [62 pull requests have been opened](https://github.com/search?q=org%3APrestaShop+is%3Apublic++-repo%3Aprestashop%2Fprestashop.github.io++is%3Apr+created%3A2019-05-20..2019-05-26) in the project repositories;
+- [54 pull requests have been closed](https://github.com/search?q=org%3APrestaShop+is%3Apublic++-repo%3Aprestashop%2Fprestashop.github.io++is%3Apr+closed%3A2019-05-20..2019-05-26), including [46 merged pull requests](https://github.com/search?q=org%3APrestaShop+is%3Apublic++-repo%3Aprestashop%2Fprestashop.github.io++is%3Apr+merged%3A2019-05-20..2019-05-26).
+
+
+## Code changes in the 'develop' branch
+
+### Core
+
+* [#13554](https://github.com/PrestaShop/PrestaShop/pull/13554): Prepares Command/Query API for migrating "Create order" page. Thank you [@sarjon](https://github.com/sarjon)
+* [#13865](https://github.com/PrestaShop/PrestaShop/pull/13865): Merge 1.7.6.x to develop - 20/05/2019, by [@matks](https://github.com/matks)
+* [#13932](https://github.com/PrestaShop/PrestaShop/pull/13932): Merge 1.7.6.x to develop - 23/05/2019, by [@matks](https://github.com/matks)
+* [#13967](https://github.com/PrestaShop/PrestaShop/pull/13967): Merge 1.7.6.x to develop - 24/05/2019, by [@matks](https://github.com/matks)
+
+
+### Front
+
+* [#13804](https://github.com/PrestaShop/PrestaShop/pull/13804):  Fix #9763 smarty compilation ID multi shop / multi theme. Thank you [@202-ecommerce](https://github.com/202-ecommerce)
+
+
+## Code changes in the "1.7.6.x" branch (for v1.7.6.0)
+
+### Core
+
+* [#13885](https://github.com/PrestaShop/PrestaShop/pull/13885): Force array cache for doctrine, by [@jolelievre](https://github.com/jolelievre)
+
+
+### Back office
+
+* [#13690](https://github.com/PrestaShop/PrestaShop/pull/13690): Theme and logo page design fixes. Thank you [@tomas862](https://github.com/tomas862)
+* [#13797](https://github.com/PrestaShop/PrestaShop/pull/13797): Adds missing legacy links for currency controller. Thank you [@tomas862](https://github.com/tomas862)
+* [#13813](https://github.com/PrestaShop/PrestaShop/pull/13813): Fixes toggling Category status in Monitoring page. Thank you [@sarjon](https://github.com/sarjon)
+* [#13840](https://github.com/PrestaShop/PrestaShop/pull/13840): Fix resetting sort and items per page on list filters reset action. Thank you [@zuk3975](https://github.com/zuk3975)
+* [#13847](https://github.com/PrestaShop/PrestaShop/pull/13847): Delete legacy AdminManufacturer controller. Thank you [@zuk3975](https://github.com/zuk3975)
```

Comment:
PRs #13910 #13852 #13850 #13849 #13847 have been reverted in https://github.com/PrestaShop/PrestaShop/pull/13956 because of legacy migration issues. I suggest we do not list them

Reply:
that will be reflected in next week core weekly